platforms/web/public/locales/en/menu.json

```diff
@@ -1,6 +1,6 @@
 {
   "close_menu": "Close menu",
-  "language_menu": "Open language menu",
+  "language_menu": "Language menu",
```

Comment:
Yeah, I initially thought so as well! But, just like we did with the show/hide Password button, because we use `aria-expanded` which will read collapsed/expanded, this is clear enough and also in line with the Password button for example

Reply:
Hm, but then it should remain "Open language menu" (combined with the `aria-expanded` state), just like "View password" (that doesn't change to "Hide password")

Reply:
Makes sense, I will revert this change!